R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
@daroczig Nice to see the follow through. =)  

Maybe you already tried this and it didn't work, but `.jpackage(AWR)` should work (and would be more clear).  It seems to me that the default is to load all jars in the `java` directory of the named package.  

You have a typo in the package documentation where the initial a in `JAva` is capitalized.  

I recommend building out a vignette (seems to increase the likelihood of garnering a mention on blogs, and after all if nobody knows about your package, it provides less value to the community).

Other than that, looks good to me.